src/Dropdown.elm

```diff
@@ -35,44 +36,54 @@ type DropdownType
     | MultiSelect
 
 
-type alias InternalState item =
+type alias InternalState =
     { id : String
     , isOpen : Bool
-    , selectedItems : List item
```

Comment:
I think it would work in simple cases like an isolated dropdown, but I think it has a similar problem as passing the default in the init function, which is that it in complicated cases where you could want to reset the default based on changes to the model, like in the case of resetting the city dropdown when a different country is selected, it would be difficult to manage as you would have to call this function every time model is updated, and therefore the developer would effectively be managing the internal state of dropdown and be responsible for duplicating the selected items.